apps/native/.env.example

```
@@ -0,0 +1,3 @@
+EXPO_PUBLIC_SUPABASE_URL=
+EXPO_PUBLIC_SUPABASE_PUBLISHABLE_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The EXPO_PUBLIC_SUPABASE_PUBLISHABLE_KEY key should go before the EXPO_PUBLIC_SUPABASE_URL key

(UnorderedKey)